src/main/scala/org/opennetworkinsight/ProxyPreLDA.scala

```diff
@@ -0,0 +1,100 @@
+package org.opennetworkinsight
+
+
+import org.apache.log4j.{Level, Logger => apacheLogger}
+import org.apache.spark.rdd.RDD
+import org.apache.spark.sql.SQLContext
+import org.apache.spark.{SparkConf, SparkContext}
+import org.slf4j.{Logger, LoggerFactory}
+import org.apache.spark.sql.types._
+import org.apache.spark.storage.StorageLevel
+
+/**
+  * Contains routines for creating the "words" for a suspicious connects analysis from incoming proxy records.
+  */
+
+object ProxyPreLDA {
+
+  def getColumnNames(input: Array[String], sep: Char = ','): scala.collection.mutable.Map[String, Int] = {
+    val columns = scala.collection.mutable.Map[String, Int]()
+    val header = input.zipWithIndex
+    header.foreach(tuple => columns(tuple._1) = tuple._2)
+    columns
+  }
+
+  def proxyPreLDA(inputPath: String, feedbackFile: String, duplicationFactor: Int,
+                  sc: SparkContext, sqlContext: SQLContext, logger: Logger): RDD[String] = {
+
+    val logger = LoggerFactory.getLogger(this.getClass)
+    apacheLogger.getLogger("org").setLevel(Level.OFF)
+    apacheLogger.getLogger("akka").setLevel(Level.OFF)
```

Comment:
I don't think we need this code - either. Logger is being passed to proxyPostLDA. 
Also apacheLogger doesn't need to be re-configured/turned off